vm/address/Cargo.toml

```diff
@@ -15,3 +15,7 @@ data-encoding-macro = "0.1.7"
 leb128 = "0.2.1"
```

Comment:
I can't comment on this since out of scope, but can you bump the version on the address crate to 0.2.3 so I can release without having to PR to bump the version?